src/Symfony/Bridge/Doctrine/DependencyInjection/CompilerPass/RegisterMappingsPass.php

```diff
@@ -234,7 +233,10 @@ private function getManagerName(ContainerBuilder $container)
             }
         }
 
-        throw new ParameterNotFoundException('Could not determine the Doctrine manager. Either Doctrine is not configured or a bundle is misconfigured.');
+        throw new \InvalidArgumentException(sprintf(
```

Comment:
1. This is potentially a BC break if anybody was depending on the `ParameterNotFoundException` for any reason.
2. Should this be a `DependencyInjection\Exception\InvalidArgumentException` or similar?